README.md

```diff
@@ -7,27 +7,77 @@ test suite, Tempest.
 
 The tempest snap can be installed directly from the snap store:
 
-    sudo snap install --edge tempest
+    sudo snap install [install-OPTIONS] tempest
+
+## Plugins
+
+The snap provides all the plugins listed in the [OpenStack release notes] and those
+cannot be removed or updated. Currently, it is not possible to add new plugins or use
+your own.
+
+List of plugins:
+
+- barbican-tempest-plugin
+- blazar-tempest-plugin
+- cinder-tempest-plugin
+- cloudkitty-tempest-plugin
+- cyborg-tempest-plugin
+- designate-tempest-plugin
+- ec2api-tempest-plugin
+- freezer-tempest-plugin
+- glance-tempest-plugin
+- heat-tempest-plugin
+- ironic-tempest-plugin
+- keystone-tempest-plugin
+- kuryr-tempest-plugin
+- magnum-tempest-plugin
+- manila-tempest-plugin
+- mistral-tempest-tests
+- monasca-tempest-plugin
+- murano-tempest-plugin
+- neutron-tempest-plugin
+- octavia-tempest-plugin
+- oswin-tempest-plugin
+- sahara-tests
+- senlin-tempest-plugin
+- solum-tempest-plugin
+- telemetry-tempest-plugin
+- trove-tempest-plugin
+- venus-tempest-plugin
+- vitrage-tempest-plugin
+- watcher-tempest-plugin
+- zaqar-tempest-plugin
+- zun-tempest-plugin
 
 
 ## Quickstart
-To run Tempest, please follow the steps in [Tempest QuickStart](https://docs.openstack.org/tempest/latest/overview.html#quickstart) documentation.
+To run Tempest, please follow the steps in the [Tempest QuickStart] documentation.
 
-The tempest snap provides a set of pre-defined test lists with different focuses. For quickly verifying the behavior of the objective cloud and making sure tempest is correctly set up, you can run tests listed in the `readonly-quick.txt` file:
+The tempest snap provides a set of pre-defined test lists with different focuses.
+For quickly verifying the behavior of the target cloud and making sure tempest is
+correctly set up, you can run tests listed in the `readonly-quick.txt` file:
 
     tempest run --load-list '@BUILTIN_TESTLISTS/readonly-quick.txt' 
 
-`@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path of the directory storing the built-in test lists.
+The `@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path
+of the directory storing the built-in test lists.
 
-**Note**: the single-quotation marks around the test list file path is required if `@BUILTIN_TESTLISTS` keyword is used (in another words, when running tempest with a built-in test list file). Fail to do that will result in wrong path error.
+**Note**: the single-quotation marks around the test list file path is required
+if the `@BUILTIN_TESTLISTS` keyword is used (in other words, when running tempest
```

Comment:
```suggestion
If the `@BUILTIN_TESTLISTS` keyword is used (in other words, when running tempest
```

Reply:
As discussed elsewhere, `if` should be kept lower-cased.